src/main/java/nextstep/subway/path/domain/FarePolicy.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.subway.path.domain;
+
+import static nextstep.subway.path.domain.AgePolicy.*;
+import static nextstep.subway.path.domain.DistancePolicy.*;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import nextstep.subway.line.domain.Line;
+
+public class FarePolicy {
+
+	private final int DEDUCTIBLE_AMOUNT = 350;
```

Comment:
상수는 static 을 붙여주면 어떨까요? 😄 
```suggestion
	private static final int DEDUCTIBLE_AMOUNT = 350;
```